t/14-isotovideo.t

```diff
@@ -291,6 +291,12 @@ subtest 'upload assets on demand even in failed jobs' => sub {
     like $log, qr/qemu-img.*foo.qcow2/, 'requested image is published even though the job failed';
     ok(-e $pool_dir . '/assets_public/foo.qcow2', 'published image exists');
     ok(!-e bmwqemu::STATE_FILE, 'no fatal error recorded') or die path(bmwqemu::STATE_FILE)->slurp;
+
+    subtest 'upload assets when publish_hdd is none' => sub {
```

Comment:
I think this subtest inside the `upload assets on demand even in failed jobs` subtest is misleading. This one is not about failed jobs.

Also I think we might be able to test this without doing a whole isotovideo call (which seems to increase the runtime by 10 seconds).
In line 327 there is a subtest about publishing assets, which unit-tests `handle_generated_assets`. Maybe that's a better place.